README.md

```diff
@@ -120,49 +120,49 @@ However, often we don't even realize that the code being tested is using the net
 
 ## Disk access
 
-Unit tests ideally should not rely on the current filesystem to run successfully. If they do, they are more error prone as the filesystem can change, and are also slower if they write stuff to the disk.
+Unit tests should ideally not rely on the current filesystem to run successfully. If they do, they are slower as well as more error prone as the filesystem can change.
```

Comment:
I don't like this change:
* it just says they are slow, not why (because of writing to disk)
* `as well as more error prone as` -> three time `as` in a short span

Reply:
Fair enough :-)